action.yaml

```
@@ -101,10 +104,15 @@ runs:
           --entrypoint "" \
           "homeassistant/home-assistant:${{ steps.version.outputs.version }}" \
             python -m homeassistant --version
-
+        if [[ -f "${{ inputs.env_file }}" ]]; then
+          env_file_arg="--env-file ${{ inputs.env_file }}"
+        else
+          env_file_arg=""
+        fi
```

Small tweak to eliminate unneeded else.

```suggestion
        env_file_arg=""
        if [[ -f "${{ inputs.env_file }}" ]]; then
          env_file_arg="--env-file ${{ inputs.env_file }}"
        fi
```